src/msg/async/AsyncMessenger.cc

```diff
@@ -580,7 +339,16 @@ int AsyncMessenger::bind(const entity_addr_t &bind_addr)
 
   // bind to a socket
   set<int> avoid_ports;
-  int r = processor.bind(bind_addr, avoid_ports);
+  int r = 0;
+  unsigned i = 0;
+  for (auto &&p : processors) {
+    r = p->bind(bind_addr, avoid_ports);
+    if (r < 0) {
+      assert(i == 0);
```

Comment:
why it is unexpected if the `rebind()` fails on the core other than the first one?